openstates/wa/committees.py

```diff
@@ -23,23 +29,26 @@ def scrape(self, chamber, term):
                 continue
 
             name = xpath(comm, "string(wa:Name)")
-            comm_id = xpath(comm, "string(wa:Id)")
+            # comm_id = xpath(comm, "string(wa:Id)")
             # acronym = xpath(comm, "string(wa:Acronym)")
-            phone = xpath(comm, "string(wa:Phone)")
+            # phone = xpath(comm, "string(wa:Phone)")
```

Comment:
I'm wondering if we can keep some of these attributes in `extras`, or maybe add them to OCD/pupa. WDYT @jamesturk ?